plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -35,4 +45,162 @@ def ProxyShapePayloads(proxyShape, loadAll):
+                dirtyStack = True
+            elif newFilePath is None:
+                debugMessage('  Empty stage, opening file dialog...')
+                openFileDialog = True
+
+        # If we have a dirty layer stack, display dialog to confirm new loading.
+        if dirtyStack:
+            kTitleFormat = mel.eval('getMayaUsdString("kDiscardStageEditsTitle")')
+            kMsgFormat = mel.eval('getMayaUsdString("kDiscardStageEditsLoadMsg")')
+            kYes = mel.eval('getMayaUsdString("kButtonYes")')
+            kNo = mel.eval('getMayaUsdString("kButtonNo")')
+            kTitle = cmds.format(kTitleFormat, stringArg=stageName)
+            # Our title is a little long, and the confirmDialog command is not making the
+            # dialog wide enough to show it all. So by telling the message string to not
+            # word-wrap, the dialog is forced wider.
+            kMsg = "<p style='white-space:pre'>" + cmds.format(kMsgFormat, stringArg=stageName)
+            res = cmds.confirmDialog(title=kTitle,
+                                     message=kMsg, messageAlign='left', 
+                                     button=[kYes, kNo], defaultButton=kYes, cancelButton=kNo, dismissString=kNo,
+                                     icon='warning')
+            if res == kYes:
+                debugMessage('  User confirmed load action, opening file dialog...')
+                openFileDialog = True
+            else:
+                # User said NO to update, so don't proceed any farther.
+                return False
+
+        if openFileDialog and newFilePath is None:
+            # Pop the file open dialog for user to load new usd file.
+            title = mel.eval('getMayaUsdString("kLoadUSDFile")')
+            okCaption = mel.eval('getMayaUsdString("kLoad")')
+            fileFilter = mel.eval('getMayaUsdString("kAllUsdFiles")') + ' (*.usd *.usda *.usdc *.usdz );;*.usd;;*.usda;;*.usdc;;*.usdz';
+            res = cmds.fileDialog2(caption=title, fileMode=1, ff=fileFilter, okc=okCaption)
+            if res and len(res) == 1:
+                debugMessage('    User picked USD file, setting file path attribute')
+                # Simply set the file path attribute. The proxy shape will load the file.
+                usdFileToLoad = res[0]
+                cmds.setAttr(filePathAttr, usdFileToLoad, type='string')
+                return True
+        elif newFilePath is not None:
+            # Instead of opening a file dialog to get the USD file, simply
+            # use the one provided as input.
+            # Note: it is important to check "not None" since the new file path can be
+            #       an empty string.
+            debugMessage('  New file to load provided, setting filepath="%s"' % newFilePath)
+            cmds.setAttr(filePathAttr, newFilePath, type='string')
+            return True
+    except Exception as e:
+        debugMessage('ProxyShapeFilePathChanged() - Error: %s' % str(e))
```

Comment:
You'll see I added this in the other functions above that had try/except blocks already. During my dev work whenever there was an error during the script it would end up here, but I didn't know why until I added the exception error message.